packages/ui/src/primitives/tailwind/Modal/index.tsx

```diff
@@ -123,7 +124,9 @@ export const ModalFooter = ({
 const Modal = ({
   id,
   title,
-  onClose,
+  onClose = () => {
```

Comment:
we should avoid having primitives importing state if possible

Reply:
Almost all modals has the same function set to `onClose`. Hence I set it as default prop. Shall I undo it?

Reply:
noted.